### DISCUSSION TITLE
handbook/enterprise_security/devsecops/process_and_landing/security_training/

### DISCUSSION THREAD

**Discussion by giscus** [SEC-CAFE/handbook#3] [General]
# handbook/enterprise_security/devsecops/process_and_landing/security_training/

企业网络安全培训与意识提升指南与实践，相关课程与演示文档

https://sec.cafe/handbook/enterprise_security/devsecops/process_and_landing/security_training/

**Comment by user1430** [comment]
对提高安全培训效果的几个建议：
1. 不对单次安全培训的内容宣贯抱太高预期，听课走神是正常的现象，低频的非生产相关培训内容无法使员工形成深刻记忆以及约束；
2. 安全培训作为安全宣导活动，培训课程和培训内容是中间环节，前置的沟通宣传环节与后置的调研回访环节都是非常好的建联窗口，把握建联机会说服关键人参与关键动作可能更具有实际落地价值；
3. 安全知识存在理解门槛，对研发人员与非研发人员都普遍适用的安全培训课程是比较有编写门槛的，如果没有编制过相关内容，不妨进行复盘案例分析，宣贯名词除了完成培训任务几乎无实际效果；
